assets/js/blocks/cart-checkout/cart/edit.js

```diff
@@ -18,13 +23,59 @@ import EmptyCart from './empty-cart';
 /**
  * Component to handle edit mode of "Cart Block".
  */
-const CartEditor = ( { className } ) => {
+const CartEditor = ( { className, attributes, setAttributes } ) => {
+	const { isShippingCalculatorEnabled, isShippingCostHidden } = attributes;
+
 	const [ isFullCartMode, setFullCartMode ] = useState( true );
 
 	const toggleFullCartMode = () => {
 		setFullCartMode( ! isFullCartMode );
 	};
 
+	const getBlockSettings = () => (
+		<InspectorControls>
+			<PanelBody
+				title={ __(
+					'Shipping calculations',
+					'woo-gutenberg-products-block'
+				) }
+			>
+				<ToggleControl
+					label={ __(
+						'Shipping calculator',
+						'woo-gutenberg-products-block'
+					) }
+					help={ __(
+						'Allow customers to estimate shipping',
+						'woo-gutenberg-products-block'
+					) }
+					checked={ isShippingCalculatorEnabled }
+					onChange={ () =>
+						setAttributes( {
+							isShippingCalculatorEnabled: ! isShippingCalculatorEnabled,
+						} )
+					}
+				/>
+				<ToggleControl
+					label={ __(
+						'Hide shipping costs',
```

Comment:
Agree with @Aljullu about the two settings being both positive (both "show").

However I think these settings are quite confusing, and getting the naming and help text right is quite challenging. cc @garymurray 

If the settings do map to core settings, then we should look at using similar wording (I haven't compared the wording).